src/ir/context.rs

```diff
@@ -630,15 +634,29 @@ impl<'ctx> BindgenContext<'ctx> {
                 // argument names don't matter in the global context.
                 if (declaration.kind() == CXCursor_ClassTemplate ||
                     declaration.kind() ==
-                    CXCursor_ClassTemplatePartialSpecialization) &&
+                    CXCursor_ClassTemplatePartialSpecialization ||
+                    declaration.kind() == CXCursor_TypeAliasTemplateDecl) &&
```

Comment:
It seems we would benefit from a `Cursor::is_template()` method here, that we would use like `declaration.is_template()` and replace all these kind comparisons.

As it is right now, this condition is super intense.